bsp/esp32_p4_function_ev_board/include/bsp/esp32_p4_function_ev_board.h

```diff
@@ -58,12 +68,17 @@
 #endif
 
 /* uSD card */
-#define BSP_SD_D0             (GPIO_NUM_39)
-#define BSP_SD_D1             (GPIO_NUM_40)
-#define BSP_SD_D2             (GPIO_NUM_41)
-#define BSP_SD_D3             (GPIO_NUM_42)
-#define BSP_SD_CMD            (GPIO_NUM_44)
-#define BSP_SD_CLK            (GPIO_NUM_43)
+#define BSP_SD_SLOT_0_DEFAULT_INIT     \
+    .clk = 0,                          \
+    .cmd = 0,                          \
+    .d0  = 0,                          \
+    .d1  = 0,                          \
+    .d2  = 0,                          \
+    .d3  = 0,                          \
```

Comment:
Yes, this part has been tested. Since slot_0 was selected in the configuration and the pin is fixed, you can simply fill in 0. In fact, only this initialization method can avoid conflicts with esp-hosted.

Reply:
Oh, I see. Thank you very much for fixing this issue. I will resume using the correct pin numbers moving forward.